scrapper.py

```diff
@@ -35,7 +48,33 @@ def find_articles(self):
         """
         Finds articles
         """
-        pass
+        headers = {
+            'user-agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64)AppleWebKit/'
+                          '537.36 (KHTML, like Gecko) Chrome/99.0.4844.51 Safari/537.36',
+            'accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,image/'
+                      'avif,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.9',
+            'accept-encoding': 'gzip, deflate, br',
+            'accept-language': 'ru,ru-RU;q=0.9,en-US;q=0.8,en;q=0.7,la;q=0.6'}
+
+        for url in self.seed_urls:
+            response = requests.get(url, headers=headers)
+
+            if not response.ok:
+                print("Request failed")
+
+            soup = BeautifulSoup(response.text, 'lxml')
+
+            articles = soup.find('div', class_='entry-content')
+            all_links = articles.find_all('a')[1:]
+            for link in all_links:
+                try:
+                    if (('https://vestnik.lunn.ru/' in link['href']) or
+                            ('http://vestnik.lunn.ru/' in link['href'])):
+                        self.urls.append(link['href'])
+                    else:
+                        self.urls.append('https://vestnik.lunn.ru/' + link['href'])
+                except KeyError:
```

Comment:
instead of making such a huge body of try/except, you need to surround the fragile part with it. what is the fragile part in your case? can you extract it to a smaller try/except with the following less erroneous code